jardin/model.py

```diff
@@ -491,13 +490,8 @@ def _default_table_name(self):
 
     @classmethod
     def db(self, role='replica', db_name=None):
-        if not hasattr(self, '_db'): self._db = {}
         name = db_name or self.db_names.get(role)
-        
-        if name not in self._db:
-            self._db[name] = DatabaseConnections.connection(name)
```

Comment:
No need to cache connection in model class. The connection object is already cached in DatabaseConnections.